var/spack/repos/builtin/packages/py-aioitertools/package.py

```diff
@@ -14,8 +14,11 @@ class PyAioitertools(PythonPackage):
     homepage = "https://aioitertools.omnilib.dev/en/stable/"
     pypi = "aioitertools/aioitertools-0.7.1.tar.gz"
 
+    version("0.11.0", sha256="42c68b8dd3a69c2bf7f2233bf7df4bb58b557bca5252ac02ed5187bbc67d6831")
     version("0.7.1", sha256="54a56c7cf3b5290d1cb5e8974353c9f52c677612b5d69a859369a020c53414a3")
 
     depends_on("python@3.6:", type=("build", "run"))
+    depends_on("python@3.6:3.9", type=("build", "run"), when="@0.11.0:")
```

Comment:
Where do you see this constraint?

Reply:
Ah! Looks like I misunderstood the requires block in the pyproject.toml.

https://github.com/omnilib/aioitertools/blob/53e15e38c4e1b5fea4876fa5f35883f11de015ff/pyproject.toml#L11